Source/EventFlow/Extensions/SpecificationExtensions.cs

```diff
@@ -34,7 +34,7 @@ namespace EventFlow.Extensions
 {
     public static class SpecificationExtensions
     {
-        public static void ThrowDomainErrorIfNotStatisfied<T>(
+        public static void ThrowDomainErrorIfNotSatisfied<T>(
```

Comment:
Please create an overload with the corrected name and put a `[Obsolete]` attribute on this as people are already using it